infrastructure/index.ts

```diff
@@ -294,56 +276,6 @@ if (config.registryStack) {
     const registryStack = new pulumi.StackReference(config.registryStack);
     const registryCDN = registryStack.getOutput("cloudFrontDomain");
 
-    bundleOrigins.push(
```

Comment:
you can probably  remove this line as well that concats the bundles origins to the overall origins array in the distribution args.
https://github.com/pulumi/docs/pull/10137/files#diff-32b0c3347657743da0ef35b9a3f3d8eca1191c2d669458dd8bfd202c0eeda87dR364

and this line also that adds the behaviors:
https://github.com/pulumi/docs/pull/10137/files#diff-32b0c3347657743da0ef35b9a3f3d8eca1191c2d669458dd8bfd202c0eeda87dL446

Reply:
Done.